doc/developers/developing_scikit.rst

```diff
@@ -0,0 +1,761 @@
+.. _developing_scikit:
+
+============
+Developing scikit-learn
+============
+
```

Comment:
"Whether you are proposing an estimator for inclusion in scikit-learn, developing a separate scikit-learn package, or implementing custom components for your own projects, this chapter details how to develop objects that safely interact with scikit-learn Pipelines and model selection tools."